datalad/support/tests/test_json_py.py

```diff
@@ -39,6 +39,17 @@ def test_load_screwy_unicode(fname):
         assert_in('Failed to decode content', cml.out)
 
 
+@with_tempfile(content=u"""\
+{"key0": "a b"}
+{"key1": "plain"}""".encode("utf-8"))
+def test_load_unicode_line_separator(fname):
+    # See gh-3523.
+    result = list(load_stream(fname))
+    print(result)
```

Comment:
Oops, stray `print`.  Tests look good, so I'll force push to remove this and then merge.

https://ci.appveyor.com/project/mih/datalad/builds/25936222
https://travis-ci.org/datalad/datalad/builds/557837997